web-controller/src/main/java/com/oasis/constant/ErrorCodeAndMessage.java

```diff
@@ -7,4 +7,10 @@
     public static final String[] PASSWORD_DOES_NOT_MATCH = {
```

Comment:
change from array of string to object
ErrorCode {
String code;
String message;
}